### PR TITLE
Adding Postgresql load generation script

### PR DESCRIPTION
**Pull request by oumkale**
**What this PR does / why we need it**:

**Which issue this PR fixes** format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
Adding Postgresql load generation script
 - cmdCommand.py getting used in CMD probe to verify the poastgres
 - loadGen.py generating query load against postgres DB 
 - schema.py common file for reusable functions and getting secrets

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -346,6 +346,16 @@ _push_litmus_kafka_deployer:
...
 
+_build_litmus_postgres_load_script:
+	@echo "INFO: Building container image for litmuschaos/postgres-load-script"
+	cd custom/workflow-helper/postgresql-helper/load-script/buildscript && docker build -t litmuschaos/postgres-load-script .
+
+_push_litmus_postgres_load_script:
+	@echo "INFO: Publish container litmuschaos/postgres-load-script"
+	cd custom/workflow-helper/postgresql-helper/load-script/buildscript && ./push
+
+litmus-postgres-load-script: deps _litmus_postgres_load_script _push_litmus_postgres_load_script
```
Comment (ispeakc0de) [inline Makefile:357]:
```suggestion
litmus-postgres-load-script: deps _build_litmus_postgres_load_script  _push_litmus_postgres_load_script
```

File: custom/workflow-helper/postgresql-helper/load-script/cmdCommand.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import base64
+
+from kubernetes import client, config
+if os.getenv('KUBERNETES_SERVICE_HOST'):
+    configs = config.load_incluster_config()
+else:
+    configs = config.load_kube_config()
+v1 = client.CoreV1Api()
+try:
+    secret = v1.read_namespaced_secret("postgres-application.credentials", "litmus")
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgresql-helper/load-script/cmdCommand.py:15]: make ns as tunable and secret name as a global variable or const

File: custom/workflow-helper/postgresql-helper/load-script/loadGen.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    print("[Error]: Unable to find secrets in litmus namespace err: ", exp)
+
+t_host = base64.b64decode(secret.data["host"]).decode('utf-8').replace('\n', '')
+t_port = base64.b64decode(secret.data["port"]).decode('utf-8').replace('\n', '')
+t_dbname = base64.b64decode(secret.data["dbname"]).decode('utf-8').replace('\n', '')
+t_name_user = base64.b64decode(secret.data["username"]).decode('utf-8').replace('\n', '')
+t_password =  base64.b64decode(secret.data["password"]).decode('utf-8').replace('\n', '')
+
+# DBDetails is connecting to a specific database and performing read and write operations
+class DBDetails(object):
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgresql-helper/load-script/loadGen.py:27]: Can we reuse common funcs?

File: custom/workflow-helper/postgresql-helper/load-script/loadGen.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        db.db_cursor.execute(s)
+        db.db_conn.commit()
+        print("[Info]: Table {} has been dropped".format(t_name_tbl))
+
+
+def Main():
+    print("[Info]: Postgres Load-testing has been started!!")
+    print("[Info]: Postgres details dbname: {}, host: {}, port: {}, username: {}".format(t_dbname, t_host, t_port, t_name_user))
+    while True:
+        time.sleep(1)
```
Comment (ispeakc0de) [inline custom/workflow-helper/postgresql-helper/load-script/loadGen.py:68]: we can move it in end of while loop

File: custom/workflow-helper/postgresql-helper/load-script/cmdCommand.py (hunk 2 above)
Comment (oumkale) [reply custom/workflow-helper/postgresql-helper/load-script/cmdCommand.py:15]: Thank you @ispeakc0de, I'm updating it

File: custom/workflow-helper/postgresql-helper/load-script/loadGen.py (hunk 4 above)
Comment (oumkale) [reply custom/workflow-helper/postgresql-helper/load-script/loadGen.py:68]: We need it for `except` condition also so have added in both try and except

File: .github/workflows/push.yml [hunk 5]
```diff
@@ -546,6 +549,19 @@ jobs:
...
         run: make fio
+
+  litmus-pg-load:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.litmus-pg-load == 'true'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```
Comment (uditgaurav) [inline .github/workflows/push.yml:560]:
```suggestion
```

we don't need this for push

File: Makefile [hunk 6]
```diff
@@ -365,6 +365,16 @@ _push_litmus_kafka_deployer:
 
 litmus-kafka-deployer: deps _build_litmus_kafka_deployer _push_litmus_kafka_deployer
 
+_build_litmus_pg_load:
+	@echo "INFO: Building container image for litmuschaos/litmus-pg-load"
+	cd custom/workflow-helper/postgres-helper/load-test && docker build -t litmuschaos/litmus-pg-load .
+
+_push_litmus_pg_load:
+	@echo "INFO: Publish container litmuschaos/litmus-pg-load"
+	cd custom/workflow-helper/postgres-helper/load-test/buildscripts && ./push
```
Comment (uditgaurav) [inline Makefile:374]:
```suggestion
	cd custom/workflow-helper/postgres-helper/load-test && ./buildscripts/push
```

Can you check the above suggestion

**Review by uditgaurav — CHANGES_REQUESTED** [review]

**Review by uditgaurav — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**